### ISSUE TITLE
Remove boxer

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#26324]
Worst RP role in the game easily, see https://github.com/space-wizards/space-station-14/issues/26323 for similar issue but Boxer is definitely worse.
- Originally rane gave it powercrept boxing gloves for reasons.
- Just fight someone in the ring (which is what most people do anyway) why does it need an entire-ass role.
- Make some actually good content for departments and not filler.

**Comment by deltanedas** [comment]
how is a boxing glove powercrept or did it used to spawn with rigged-like gloves

**Comment by metalgearsloth** [comment]
> how is a boxing glove powercrept or did it used to spawn with rigged-like gloves

Boxer (the role) did a lot of stun / stamina damage, then we ended up moving it to the gloves iirc.

**Comment by asperger-sind** [comment]
> how is a boxing glove powercrept or did it used to spawn with rigged-like gloves

Boxing gloves right now aren't powercrept at all, in fact they're weak. Boxer never spawned with rigged gloves iirc.

What Sloth is talking about looks like BoxerComponent which made you deal like 1.75x the damage with boxing gloves but was since then removed.

**Comment by ElPidor** [comment]
This role may be removed but its very funny instead of making roles really interesting just deleting all shit making a silly excuse 
Just make boxer be a fucking martal artist and any other shit will go away, 
Give him block or some simple combos with gloves and here we go an actuall rp role, kek

**Comment by Hmeister-real** [comment]
yeah, why the hell are you and the other maintainers trying to entirely remove roles rather than reworking/improving them? Its not as if they are broken or widely hated, just a little stale since the boxer and musician are relatively unchanged since their original conception as a role

> this is gonna be a repeat of what happened with removing space pens or the removal of senior roles, isnt it? 😿

**Comment by metalgearsloth** [comment]
> yeah, why the hell are you and the other maintainers

calm down mate.

> > this is gonna be a repeat of what happened with removing space pens or the removal of senior roles, isnt it? 😿

There's currently open PRs.

**Comment by FairlySadPanda** [comment]
So on Boxer I think there's just a theme problem here tbh that makes it waaay too limited a role.

A service role that exists to promote crew excercise/health/self-defence training would justify boxing rings, gyms and other rooms for space stations without being specifically labelled "boxer".

It could be RP'd in more ways, it could be used as a chill training/robustness practicing role (a newbie wants to know how to defend themselves - train them!) and someone could literally just put on boxing gloves and run around shouting FIGHT ME like currently.

**Comment by IProduceWidgets** [comment]
1984 it, role can just be a loadout of passenger when / if we get some sort of loadout reward system.

**Comment by ChaseFlorom** [comment]
I actually like @IProduceWidgets  suggestion the best. I do think some of the RP roles just clutter things up and don't often provide a ton of value, but either do passengers. Having some sort of loadout system for passengers could be cool and incentivize  more RP on their part.

**Comment by Cadunkus** [comment]
If absolutely everything in your job can be done by a passenger at the very start of the round, then that job has no reason to exist.

**Comment by Djungelskog2** [comment]
> If absolutely everything in your job can be done by a passenger at the very start of the round, then that job has no reason to exist.

ok yea just take a visit to the HoP and we have reason to remove all of:
Cargo tech
Salvage
Atmosian
Engineer
Chemist
Doctor
Paramedic
Scientist
Bartender
Botanist
Chaplain
Chef
Clown
Janitor
Lawyer
Librarian
Musician
Service worker
Zookeeper
Psychologist
Reporter
Boxer (like you said)
Don't Get me started on what you can get if you take extra time to get mind shielded too

**Comment by EmoGarbage404** [comment]
How can a passenger do an atmos tech's job roundstart if they have to get access from HOP and equipment from inside engineering? Are you even reading their argument or are you just having a kneejerk reaction to something you disagree with?

**Comment by Djungelskog2** [comment]
> How can a passenger do an atmos tech's job roundstart if they have to get access from HOP and equipment from inside engineering? Are you even reading their argument or are you just having a kneejerk reaction to something you disagree with?

What argument? 
Are they detracting from their own statement? because you don't have boxing gloves roundstart as a passenger

**Comment by EmoGarbage404** [comment]
Considering boxing gloves are mapped openly in rings on most maps, they are pretty undeniably able to accessed roundstart.

**Comment by Djungelskog2** [comment]
They said a similar thing on the musician PR which actually DOES need access so that's what I was going off of 
(I don't particularly care about boxer but I do the musician)

**Comment by Cadunkus** [comment]
> They said a similar thing on the musician PR which actually DOES need access so that's what I was going off of, I couldnt help but feel like this statement was intended to be applied to their other one about musician too. (I don't particularly care about boxer but I do the musician)

And you can still play musician and boxer when they remove the jobs because they exist as _roles_ not actual jobs. Go to a server without musician as jobs on the crew roster but that still has instruments in the bar and I guarantee you some assistant is going to walk up, pick up a guitar, and play MIDIs the entire shift. Neither musician nor boxer need to exist as jobs for the roles to exist ingame and - by adding them as jobs - you're ironically killing the roles because now only the person with the ID on their hip is socially expected to use them.

This is a PR to remove the boxer **job** to enable the boxer **role**.

**Comment by Djungelskog2** [comment]
> And you can still play musician and boxer when they remove the jobs because they exist as _roles_ not actual jobs. Go to a server without musician as jobs on the crew roster but that still has instruments in the bar and I guarantee you some assistant is going to walk up, pick up a guitar, and play MIDIs the entire shift. Neither musician nor boxer need to exist as jobs for the roles to exist ingame and - by adding them as jobs - you're ironically killing the roles because now only the person with the ID on their hip is socially expected to use them.
> 
> This is a PR to remove the boxer **job** to enable the boxer **role**.

Anyone can do any job I think I get where your coming from for the boxer, especially since boxers are something that atleast 2 people should be at a time, or where people can just publicly participate and are, from what I'm aware, is the only job with no access changes from oassenger
But your other points don't make sense to me, any random passenger could jump a counter and make food or don some whacky clothes and start tossing pies or fix some wires. The job existing doesn't kill the role at all, it just gives more opportunity to the people that actually want to commit, nobody is going to bat an eye for you playing the harmonica walking around in the halls and the musician always has an instrument to spare.

What's the difference between playing some publicly available instruments and being able to throw pie in someone's face in terms of "how easy can someone random do this outside of a job"

**Comment by Cadunkus** [comment]
> boxers are something that atleast 2 people should be at a time... ...But your other points don't make sense to me, any random passenger could jump a counter and make food or don some whacky clothes and start tossing pies or fix some wires. The job existing doesn't kill the role at all...

Firstly thanks to Willow (or the training robot, depending on your server) you can practice melee singleplayer. Second that's actually why boxer and "greyshirt gimmicks given an ID" really shouldn't be jobs and should be publicly available: Because you're not supposed to jump counters and steal people's work. Since there's a boxer job, people automatically think that only the boxer should be using the ring unless invited by the existing guy with the ID.

Remove the ID (and make it a standard to put boxing gloves and shorts at the ring on every station) and security can suddenly use it to train cadets in baton usage, the clown can don mitts and the moniker "Puncho McGee", atmos techs can take their argument about how the TEG is supposed to be set up to the ring, and you can _still_ strap on the gloves and become the true title-holder of "most robust" _as a passenger_ **all at any time.**

The boxing ring being a public recreational activity is better than it being its own sub-department and the role of boxer is restrained by the job of boxer.

**Comment by Cadunkus** [comment]
Also getting a job change to boxer whenever you wanna use the ring is not feasible. That's way too much of a commitment for an RP bit.

**Comment by Djungelskog2** [comment]
> Firstly thanks to Willow (or the training robot, depending on your server) you can practice melee singleplayer. Second that's actually why boxer and "greyshirt gimmicks given an ID" really shouldn't be jobs and should be publicly available: Because you're not supposed to jump counters and steal people's work.

I was using that clown comparison as a bit of a strawman but do you actually think the clown and such should be removed too by your perspective? You still haven't clarified what you mean by vague statements like that, I'm assuming your lumping musician into that group and that would be similarly lumping the clown, mime, chaplain and other RP-centric service roles into the mix.

>Since there's a boxer job, people automatically think that only the boxer should be using the ring unless invited by the existing guy with the ID.

Nobody thinks that, nobody is enforcing these "social norms" that never existed in the first place

> Remove the ID (and make it a standard to put boxing gloves and shorts at the ring on every station) and security can suddenly use it to train cadets in baton usage, the clown can don mitts and the moniker "Puncho McGee", atmos techs can take their argument about how the TEG is supposed to be set up to the ring, and you can _still_ strap on the gloves and become the true title-holder of "most robust" _as a passenger_ **all at any time.**

You can already do this literally nothing is stopping you, and having the gloves and shorts are standard on every station with a ring.

>
> The boxing ring being a public recreational activity is better than it being its own sub-department and the role of boxer is restrained by the job of boxer.

That's literally what it already is, the only thing stopping you is your own imaginary standards or boundaries.

**Comment by Cadunkus** [comment]
> I was using that clown comparison as a bit of a strawman but do you actually think the clown and such should be removed too by your perspective? You still haven't clarified what you mean by vague statements like that, I'm assuming your lumping musician into that group and that would be similarly lumping the clown, mime, chaplain and other RP-centric service roles into the mix.

No. Other roleplay jobs like the Lawyer, Bartender, Chaplain, Clown, Mime, the librarian when someone finally adds mechanics for them because there's no paintings/books archives yet, etc. all have a workplace, equipment, and/or innate mechanics that reinforce the roleplay role. A passenger couldn't take up the role of lawyer without breaking rules. As I've said before **the job** Musician and Boxer only restrict **the role** of Musician and Boxer.

> Nobody thinks that, nobody is enforcing these "social norms" that never existed in the first place

It does! I know this from experience. Every Space Station 13 server (not including alt gamemodes or ones without instruments at all) have a thriving MIDI sharing scene simply because anyone can play an instrument but on Space Station 14 it's rare to see more than one maestro at a time if you don't include PAIs (they literally can't do anything but play MIDIs).

> You can already do this literally nothing is stopping you, and having the gloves and shorts are standard on every station with a ring.

Refer to previous point. There is actually a strong social norm in ss13/14 that only filthy greytiders jump counters to steal jobs from other people. Boxer should be publicly available so there's no metaphorical counter to jump.

> That's literally what it already is, the only thing stopping you is your own imaginary standards or boundaries.

Good! Let's reinforce that by cutting the job from the roster.

I'm under the impression you're a fan of boxer and I'm trying to get you to see this is actually a really beneficial change for that role. We're on the same page here, we both want boxer (the role) to be better and - unlike other jobs - it would actually be better to remove the boxer ID and open the ring to the public.

**Comment by Djungelskog2** [comment]
> No. Other roleplay jobs like the Lawyer, Bartender, Chaplain, Clown, Mime, the librarian when someone finally adds mechanics for them because there's no paintings/books archives yet, etc. all have a workplace, equipment, and/or innate mechanics that reinforce the roleplay role. A passenger couldn't take up the role of lawyer without breaking rules. As I've said before **the job** Musician and Boxer only restrict **the role** of Musician and Boxer.

As far as I know, the lawyer and clown have no unique abilities outside of the clowns clumsiness trait which barely qualifies and even the mime only has an invisible wall. The librarian doesn't even have much depth in 13 either! (however we do have an accepted librarian design doc for 14.)

> It does! I know this from experience. Every Space Station 13 server (not including alt gamemodes or ones without instruments at all) have a thriving MIDI sharing scene simply because anyone can play an instrument but on Space Station 14 it's rare to see more than one maestro at a time if you don't include PAIs (they literally can't do anything but play MIDIs).

I think your drawing the wrong message from this, ss13 also has a far older playerbase with more midis, it has a more in-depth midi player and I'll be real, in the couple dozen or so hours I've played I've seen people playing music at the same rate as 14.
 
> Refer to previous point. There is actually a strong social norm in ss13/14 that only filthy greytiders jump counters to steal jobs from other people. Boxer should be publicly available so there's no metaphorical counter to jump.

There are no counters to jump. It is publicly available. The ring doors are even AA generally.

> Good! Let's reinforce that by cutting the job from the roster.

Ones that only I've seen *you* have
> 
> I'm under the impression you're a fan of boxer and I'm trying to get you to see this is actually a really beneficial change for that role. We're on the same page here, we both want boxer (the role) to be better and - unlike other jobs - it would actually be better to remove the boxer ID and open the ring to the public.

Did you read my previous messages? I'm only still arguing in boxers favour because I've yet to see any real arguments as to remove it outside of "it's rolebloat and it's passenger that starts with gloves and half naked" which is true and because of other similar roles.

**Comment by Cadunkus** [comment]
> I'm only still arguing in boxers favour because I've yet to see any real arguments as to remove it outside of "it's rolebloat and it's passenger that starts with gloves and half naked" which is true and because of other similar roles.

And that alone is enough to justify killing the restriction boxer has.

**Comment by Hitlinemoss** [comment]
#41741 resolves this issue. (I don't have permissions to mark issues as closed though so someone else is going to need to do that.)
